src/index.html

```diff
@@ -5,9 +5,147 @@
+      <div class="header__phone"><span>+1 234 555-55-55</span></div>
+      <div class="header__action">
+        <a href="/" class="header__link">call to order</a>
+      </div>
+      <img src="../images/logo.png" alt="logo" class="logo header__logo">
+      <div class="header__title">Miami<br> Condo Kings</div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item_active">Home</li>
+          <li class="nav__item"><a href="/" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="/" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="/" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="/" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="/" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="section">
+        <div class="info">
+          <h1 class="info__title">Miami best<br>  Real estate <a class="info__link" href="/">here</a></h1>
+          <div class="info__text">
+            <h2 class="info__heading">Over 50 years experience in love.</h2>
+            <p class="info__article">
+                Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+                <br><br>
+                The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals.
+                <br><br>
+                <strong>We will never be late! <br>
+                Guaranteed secure transaction - 100%</strong>
+            </p>
+          </div>
+        </div>
+        <h2 class="gallery-heading">Gallery</h2>
+        <div class="gallery__photos">
+          <img src="../images/photos/1-min.jpg" alt="photo" class="gallery__photos-img gallery__photos-img--wide">
+          <img src="../images/photos/2-min.jpg" alt="photo" class="gallery__photos-img gallery__photos-img--narrow">
+        </div>
+        <p class="gallery__address"><a class="gallery__link" href="/">Marina Palms</a> / North Miami Beach, FL 33162</p>
+        <div class="gallery__photos">
+            <img src="../images/photos/3-min.jpg" alt="photo" class="gallery__photos-img">
+            <img src="../images/photos/4-min.jpg" alt="photo" class="gallery__photos-img">
+        </div>
+        <p class="gallery__address"><a class="gallery__link" href="/">Edition Residens</a> / Miami Beach, FL 33139 </p>
+        <div class="gallery__photos">
+            <img src="../images/photos/5-min.jpg" alt="photo" class="gallery__photos-img gallery__photos-img--narrow">
```

Comment:
переноси длинные теги, иначе читать сложно
рекомендуемая максимальная длина строки - 80 символов
```suggestion
            <img
              src="../images/photos/5-min.jpg"
              alt="photo"
              class="gallery__photos-img gallery__photos-img--narrow"
            >
```